packages/theme/modules/catalog/pages/product.vue

```diff
@@ -452,17 +457,50 @@ export default defineComponent({
       }
     });
 
+    const { params: { id } } = route.value;
+    const baseSearchQuery = () => ({
```

Comment:
```suggestion
    const getBaseSearchQuery = () => ({
```

This is a function, not a variable (I know you just moved the code)

Reply:
In the next comment I also mention the possibility of using a computed - in that case, this comment doesn't apply obviously

Reply:
Fixed